src/bindings/ScriptDev2/scripts/outland/black_temple/boss_illidan.cpp View
@@ -1534,6 +1529,8 @@ struct MANGOS_DLL_DECL npc_flame_of_azzinothAI : public ScriptedAI
  
  		pClosestGlaive = NULL;
  		pFarthestGlaive = NULL;
+
+		m_creature->ApplySpellImmune(0, IMMUNITY_SCHOOL, SPELL_SCHOOL_MASK_FIRE, true);
  	}
  
  	void JustSummoned(Creature* pSummoned) override